test-e2e/smoketest.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+
+const WAIT = 1000;
+const NODE_PORT = process.env.NODE_PORT || 3000;
+
+
+module.exports = {
+  'Smoketest' (browser) {
+    browser
+      .url(`http://localhost:${NODE_PORT}/test-data`)
+      .waitForElementVisible('body', WAIT)
+      .assert.containsText('body', 'Books')
```

Comment:
Is "Books" part of the sample data set, or part of the React component? If it's from the React component, we should also ensure that same of the sample data actually gets rendered. =)